src/material/chips/_chips-theme.scss

```diff
@@ -94,7 +94,8 @@
 @mixin density($config-or-theme) {
   $density-scale: theming.get-density-config($config-or-theme);
   $density-scale: theming.clamp-density($density-scale, -2);
-  .mat-mdc-chip {
+  .mat-mdc-chip,
+  .mat-mdc-standard-chip.mat-mdc-standard-chip {
```

Comment:
Doesn't `mat-mdc-chip` already target `mat-mdc-standard-chip`?

Reply:
I think so. Should I just use `.mat-mdc-chip.mat-mdc-chip` instead?

Reply:
Yeah I think that should be enough.

Reply:
Ah sorry, I replied too quickly. I think it should be `.mat-mdc-chip.mat-mdc-standard-chip`. The basic chips don't have any styles AFAIK.

Reply:
Do you mean they aren't meant to have any styles? Currently in our dev-app, they are receiving density styles because we target `.mat-mdc-chip` instead of `.mat-mdc-standard-chip`

Reply:
I think that technically they aren't supposed to have any styles.